app/product/[handle]/page.tsx

```diff
@@ -84,12 +84,18 @@ export default async function ProductPage({ params }: { params: { handle: string
       <div className="mx-auto max-w-screen-2xl px-4">
         <div className="flex flex-col rounded-lg border border-neutral-200 bg-white p-8 dark:border-neutral-800 dark:bg-black md:p-12 lg:flex-row lg:gap-8">
           <div className="h-full w-full basis-full lg:basis-4/6">
-            <Gallery
-              images={product.images.map((image: Image) => ({
-                src: image.url,
-                altText: image.altText
-              }))}
-            />
+          <Suspense
```

Comment:
@leerob i know it's old, by can i know why you surrounded the Gallery by Suspense?

Reply:
is it because the Partial Rendering?